pkg/services/alerting/notifiers/slack.go

```diff
@@ -185,13 +258,22 @@ func (sn *SlackNotifier) Notify(evalContext *alerting.EvalContext) error {
 	}
 
 	body := map[string]interface{}{
+		"text": evalContext.Rule.Message,
```

Comment:
I think you want to use `evalContext.GetNotificationTitle()` here instead to make it backward compatible.